perses/annihilation/relative.py

```diff
@@ -2629,3 +2629,1504 @@ def _handle_exceptions(self):
+                             "eps_RF = {eps_RF};",
+                             "r_cutoff = {r_cutoff};",
+                             "w_scale = {w_scale};"
+                             ]
+    # predefined RF parameters (fed to the `expression` via `format`): [scale_bool_string, old_bool_string, new_bool_string, w_scale, switching_ratio, eps_RF, r_cutoff]
+    # RF per_particle_parameters: [scale_bools, old/new_bool_string, unique_old, unique_new, core]
+
+    #electrostatic exception
+    _default_RF_exception_expr_list = copy.deepcopy(_default_RF_expr_list)
+    _default_RF_exception_expr_list[0] = "U_electrostatics * {scale_bool_string};"
+    _default_RF_exception_expr_list[5] = "chargeProd = select(1 - environment_region, chargeProd_old * {old_bool_string} + chargeProd_new * {new_bool_string}, chargeProd_old);"
+    _default_RF_exception_expr_list[6] = ''
+    _default_RF_exception_expr_list[7] = ''
+    _default_RF_exception_expr_list[10] = "w = (is_unique_old * r_cutoff * w_scale * (1 - lambda_0_electrostatics_exceptions_old)) + (is_unique_new * w_scale * (1 - lambda_0_electrostatics_exceptions_new));"
+    _default_RF_exception_expr_list[11] = "is_unique_old = step(unique_old - 0.1);"
+    _default_RF_exception_expr_list[12] = "is_unique_new = step(unique_new - 0.1);"
+
+    _default_RF_expression = ' '.join(_default_RF_expr_list)
+    _default_RF_exception_expression = ' '.join(_default_RF_exception_expr_list)
+
+    _default_steric_expr_list = ["U_sterics;",
+
+                                  "U_sterics = 4 * epsilon * x * (x - 1.0);",
+                                  "x = (sigma / r_eff)^6;",
+
+                                  # sigma
+                                  "sigma = (sigma1 + sigma2) / 2;",
+                                  "sigma1 = (sigma_old1 * {old_bool_string1}) + (sigma_new1 * {new_bool_string1});",
+                                  "sigma2 = (sigma_old2 * {old_bool_string2}) + (sigma_new2 * {new_bool_string2});",
+
+                                  # epsilon
+                                  "epsilon = sqrt(epsilon1 * epsilon2);",
+                                  "epsilon1 = ((epsilon_old1 * {old_bool_string1}) + (epsilon_new1 * {new_bool_string1})) * {scale_bool_string1};",
+                                  "epsilon2 = ((epsilon_old2 * {old_bool_string2}) + (epsilon_new2 * {new_bool_string2})) * {scale_bool_string2};",
+
+                                   #r_eff
+                                   "r_eff = sqrt(r^2 + w^2);",
+                                   
+                                   #4th dimension:
+                                   #"w = step(unique_new1 + unique_new2 + unique_old1 + unique_old2 - 0.1) * r_cutoff * ({old_bool_string1} - {new_bool_string1}) * {w_scale};",
+                                   "w = is_unique_old * r_cutoff * w_scale * (1 - lambda_0_sterics_old) + is_unique_new * w_scale * (1 - lambda_0_sterics_new);",
```

Comment:
Here as well: `r_cutoff * w_scale` appears in the first term, but only `w_scale` in the second term.